src/archivematicaCommon/lib/storageService.py

```diff
@@ -8,9 +8,13 @@
 sys.path.append("/usr/share/archivematica/dashboard")
 from main.models import DashboardSetting
 
-logger = logging.getLogger(__name__)
-logging.basicConfig(filename="/tmp/archivematica.log",
-    level=logging.INFO)
+sys.path.append("/usr/lib/archivematica/archivematicaCommon")
```

Comment:
This is another case where the rebase reintroduced some deleted code, sorry - removing!